agreement_serviceprofile/models/agreement_serviceprofile.py

```diff
@@ -2,14 +2,19 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class AgreementServiceProfile(models.Model):
     _name = 'agreement.serviceprofile'
     _description = 'Agreement Service Profiles'
 
+    def _default_stage_id(self):
+        return self.env.ref('agreement_serviceprofile.servpro_stage_draft')
+
     name = fields.Char(string="Name", required=True)
+    stage_id = fields.Many2one('agreement.stage', string="Stage",
+                               default=_default_stage_id, copy=False)
```

Comment:
```suggestion
                               default=_default_stage_id, copy=False,
                               group_expand='_read_group_stage_ids',)
                               
    @api.model
    def _read_group_stage_ids(self, stages, domain, order):
        stage_ids = self.env['agreement.stage'].search([('stage_type',
                                                   '=', 'serviceprofile')])
        return stage_ids
```

Reply:
@brian10048 That should open all the stages, even without records.

Reply:
Thank you.  I added the method, but forgot that part in the field definition!